src/runtime/entitlements-manager.js

```diff
@@ -92,6 +92,7 @@ export class AvailableIntervention extends Intervention {
     const flow = new AudienceActionFlow(this.deps_, {
       isClosable: params.isClosable,
       action: this.type,
+      configurationId: this.configurationId,
```

Comment:
@mhkawano can you confirm this change is required?

Reply:
Yes, this looks right. We will also need the config Id.